### ISSUE TITLE
Class implementation for XXX not found

### ISSUE THREAD

**Issue by sbcgua** [abaplint/abaplint#2069]
Potentially a bug ... though might be just a bad code style from my side.
https://github.com/sbcgua/mockup_loader/blob/master/src/zcl_mockup_loader_stub_base.clas.testclasses.abap#L28

So, there is class, inheriting from an abstract class. Empty implementation is OK from the compiler side. But not so from abaplint side. Even though nothing to implement in additional the the existing methods.

```abap

class lcl_mockup_loader_stub_final definition final
  inheriting from zcl_mockup_loader_stub_base
  friends ltcl_mockup_stub_base_test.
endclass.

" No implementation

```

**Comment by larshp** [comment]
starting this

**Comment by larshp** [comment]
reproduced as below, actually this gives 2 false positives

```abap
CLASS zcl_mockup_loader_stub_base DEFINITION.
ENDCLASS.

CLASS lcl_mockup_loader_stub_final DEFINITION FINAL
  INHERITING FROM zcl_mockup_loader_stub_base.
ENDCLASS.
```

**Comment by sbcgua** [comment]
:+1: :)

**Comment by larshp** [comment]
I'll release a new version in a bit, if the regression test doesn't give problems

**Comment by larshp** [comment]
try again, 2.76.6

**Comment by sbcgua** [comment]
Cool :+1: It works now: https://github.com/sbcgua/mockup_loader/runs/3203602168
Thanks a lot :)
